.envrc

```diff
@@ -1,26 +1,17 @@
 #!/usr/bin/bash
 set -e
 
-if [ "$(python -V)" != "Python $(cat .python-version)" ]; then
```

Comment:
This is too specific. Down where pdm is installed we switch from `python` to `python3` and hope that it is at least Python 3.7